### ISSUE TITLE
basis-* is invalid class

### ISSUE THREAD

**Issue by bfourgeaud** [windicss/windicss-intellisense#324]
The windicss class **basis-*** is not detected as a valid windicss class by intellisense.
The classes work as expected, but are marked as error in my stylesheet.

`.emailInfo {
  @apply basis-full tablet:(basis-1);
}`

I use .scss file.

**Comment by lowgos** [comment]
Hello guys, I've got the same problem.

I'm using Vue 3, the style tag is a `scss` lang scoped tag.

I checked that I didn't write any windi configuration that should overwrite this rule, even more the rule works: it's just a false error.

![my vue style tag](https://user-images.githubusercontent.com/22097902/159968528-4992e8df-3f16-4198-882e-6fa5c6e51e00.png)

![the error i got](https://user-images.githubusercontent.com/22097902/159968435-0a3d08fe-d2c7-4eee-8d82-a1a22e89beaf.png)
